web/nginx/nginx.conf

```diff
@@ -0,0 +1,66 @@
+worker_processes auto;
```

Comment:
This config seems overly complicated for a server that sits behind another Nginx, which has most of these settings applied.

Reply:
Simplified in new commit.